libs/azure-api-client/phpunit.xml.dist

```diff
@@ -11,7 +11,10 @@
          stopOnFailure="false"
          bootstrap="tests/bootstrap.php"
          cacheResultFile="/tmp/phpunit.result.cache"
-         printerClass="Sempro\PHPUnitPrettyPrinter\PrettyPrinterForPhpUnit9">
+         printerClass="Sempro\PHPUnitPrettyPrinter\PrettyPrinterForPhpUnit9"
+         executionOrder="random"
```

Comment:
To bych dal do háje, akorát ti to dělá peklo když chceš něco debugovat

Na takový špeky tam máme ten infection

Reply:
Chtel jsem tu machrovat, ze je to lepsi, protoze se dovis ten random seed a muzes si to zreplikovat. ALe koukam, ze i ten Infection ho tam vypise, takze jsem si to tu poustel 10 dokola zbytecne :smile: Vyhodim to